src/Sqids/SqidsEncoder.cs

```diff
@@ -288,7 +291,7 @@ public int[] Decode(ReadOnlySpan<char> id)
 	/// if the ID represents a single number); or an empty array if the input ID is null,
 	/// empty, or includes characters not found in the alphabet.
 	/// </returns>
-	public int[] Decode(string id) => Decode(id.AsSpan());
+	public T[] Decode(string id) => Decode(id.AsSpan());
```

Comment:
Since this PR targets .Net 7 only, this method can be removed entirely.

Reply:
I completely forgot about this, thanks for noticing!